src/scales/scale.timeseries.js

```diff
@@ -0,0 +1,189 @@
+'use strict';
+
+module.exports = function(Chart) {
+
+	var helpers = Chart.helpers;
+	var timeHelpers = helpers.time;
+
+	// Default config for a timeseries scale
+	var defaultConfig = {
+		position: 'bottom',
+
+		time: {
+			parser: false, // false == a pattern string from http://momentjs.com/docs/#/parsing/string-format/ or a custom callback that converts its argument to a moment
+			format: false, // DEPRECATED false == date objects, moment object, callback or a pattern string from http://momentjs.com/docs/#/parsing/string-format/
+			unit: false, // false == automatic or override with week, month, year, etc.
+			round: false, // none, or override with week, month, year, etc.
+			minUnit: 'millisecond',
+
+			// defaults to unit's corresponding unitFormat below or override using pattern string from http://momentjs.com/docs/#/displaying/format/
+			displayFormats: {
```

Comment:
Now time and timeseries scale has different displayFormats because timeseries scale does not support tick alignment and major units yet. I'm working on the next PR that adds this functionality to the timeseries scale. There display formats will be same for both scales and then I can move it to helpers.time.